RM_vs_DFN.ipynb

```diff
@@ -0,0 +1,337 @@
+{
```

Comment:
<p>The plotting is a bit crowded. I wonder if you plot against charge throughput (which is the integral of the absolute value of the current, and should be an existing PyBaMM variable) if it would be clearer. In that case, they all would be roughly same size, which might mean they overlap too much, so feel free to choose whatever looks better.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/brosaplanella/formation/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
"Great idea, I've changed it, and it looks better now.